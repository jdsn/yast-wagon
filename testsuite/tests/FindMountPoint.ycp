{

include "testsuite.ycp";

include "wagon/wagon_helpers.ycp";

//import "Wagon";
DUMP("Test nil");
TEST(``(FindMountPoint(nil, nil)), [], nil);
TEST(``(FindMountPoint(nil, [])), [], nil);
TEST(``(FindMountPoint(nil, ["/boot", "/"])), [], nil);

DUMP("Test empty string");
TEST(``(FindMountPoint("", nil)), [], nil);
TEST(``(FindMountPoint("", [])), [], nil);
TEST(``(FindMountPoint("", ["/boot", "/"])), [], nil);

DUMP("Test valid values");
TEST(``(FindMountPoint("/", ["/boot", "/", "/usr"])), [], nil);
TEST(``(FindMountPoint("/usr", ["/boot", "/", "/usr"])), [], nil);
TEST(``(FindMountPoint("/usr/", ["/boot", "/", "/usr"])), [], nil);
TEST(``(FindMountPoint("/usr/share/locale", ["/boot", "/", "/usr"])), [], nil);
TEST(``(FindMountPoint("/usr/share/locale", ["/boot", "/", "/usr", "/usr/share"])), [], nil);
}
