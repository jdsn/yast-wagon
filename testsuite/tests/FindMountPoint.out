Dump	Test nil
Return	/
Return	/
Return	/
Dump	Test empty string
Return	/
Return	/
Return	/
Dump	Test valid values
Return	/
Return	/usr
Return	/usr
Return	/usr
Return	/usr/share
